tests/js/spec/views/dashboardsV2/detail.spec.jsx

```diff
@@ -325,5 +325,52 @@ describe('Dashboards > Detail', function () {
 
       expect(modal.find('AddDashboardWidgetModal').props().widget).toEqual(widgets[0]);
     });
+
+    it('hides and shows manage dashboards based on feature', async function () {
+      wrapper = mountWithTheme(
+        <DashboardDetail
+          organization={initialData.organization}
+          params={{orgId: 'org-slug', dashboardId: '1'}}
+          router={initialData.router}
+          location={initialData.router.location}
+        />,
+        initialData.routerContext
```

Comment:
```suggestion
```

Reply:
I need to pass router and router context to the DashboardDetail view otherwise the tests fail